presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,249 @@
+``authentication.ldap.enabled``                         Enable LDAP authentication for the Presto coordinator.
+                                                        Must be set to ``true``. Default value is ``false``.
+``authentication.ldap.url``                             The url to the LDAP server. We recommend using 
+                                                        ``ldaps://`` to enable Secure LDAP over SSL.
+``authentication.ldap.server-type``                     The type of LDAP server implementation.
+                                                        Set to ``active_directory`` or ``openLDAP``.                                                        
+``authentication.ldap.ad-domain``                       The domain name of the Active Directory server.
+                                                        This property is required only if 
+                                                        ``authentication.ldap.server-type`` is set to
+                                                        ``Active_Directory``. Eg: ``corp.domain.com``.
+``authentication.ldap.base-dn``                         The base LDAP distinguished name for the user who
+                                                        who tries to connect to the server. This property is 
+                                                        required if ``authentication.ldap.server-type`` is set
+                                                        to ``openLDAP``.
+``http-server.https.enabled``                           Enables HTTPS access for the Presto coordinator.
+                                                        Should be set to ``true``. Default value is 
+                                                        ``false``.
+``http-server.https.port``                              HTTPS server port.
+``http-server.https.keystore.path``                     The location of the Java Keystore file that will be
+                                                        used to secure TLS.
+``http-server.https.keystore.key``                      The password for the keystore. This must match the
+                                                        password you specified when creating the keystore.
+======================================================= ======================================================
+
+Authorization based on LDAP Group Membership
+********************************************
+
+You can further restrict the set of users allowed to connect to the Presto
+coordinator based on their group memberships. This optional feature is enabled
+if you set ``authentication.ldap.group-dn`` property.
+Only users belonging to the group with distinguished name
+``authentication.ldap.group-dn`` are authenticated successfully.
+
+In addition to the basic LDAP authentication properties, you need group-membership specific
+properties in ``config.properties``. Sample entries are listed below.
```

Comment:
Or should you rather say "Properties that need to be added, along with their sample values, are listed below." (this applies to everywhere else too wherever this sentence "Sample entries... are listed below" appears).